action.yml

```diff
@@ -2,17 +2,17 @@ name: Changelogs generated with git-cliff
 description:  Generate Changelog with a customized template 
 author: tj-actions
 inputs:
-  args:
```

Comment:
why this was removed? ho do i pass custom args?

- https://github.com/tj-actions/git-cliff/issues/80